app/scripts/views/complete_sign_up.js

```diff
@@ -236,7 +236,12 @@ define(function (require, exports, module) {
 
         // If a user is already signed in to Sync which is different to the
         // user that just verified, show them the old "Account verified!" screen.
-        return isInExperimentGroup && ! isAnotherUserSignedIn;
+        return isInExperimentGroup &&
+               ! isAnotherUserSignedIn &&
+               // The auth server can gate whether users can send an SMS based
+               // on the user's country and whether the SMS provider account
+               // has sufficient funds.
+               verifiedAccount.smsStatus();
```

Comment:
> above could be adjusted to {Boolean|Promise} clarify this ?

It never actually returns a Boolean, only a promise, because the whole function is wrapped in a promise.